src/components/SearchGridFilter.vue

```diff
@@ -143,9 +145,12 @@ export default {
         { code: 'mccordmuseum', name: 'Montreal Social History Museum' },
         { code: 'met', name: 'Metropolitan Museum of Art' },
         { code: 'museumsvictoria', name: 'Museums Victoria' },
+        { code: 'nhl', name: 'TODO' },
```

Comment:
@aldenstpage do you know the full name of this provider?

Reply:
See #146